pabi_procurement/models/purchase_requisition.py

```diff
@@ -258,6 +267,13 @@ def rejected(self):
         self.state = 'rejected'
 
     @api.multi
+    def send_pd(self):
```

Comment:
What is "pd", do we actually use this field officially? Is it "requisition" in Odoo context? (let's discuss)